cmd/podman/networks/list.go

```diff
@@ -8,7 +8,9 @@ import (
 	"text/tabwriter"
 	"text/template"
 
+	"github.com/containers/podman/v2/cmd/podman/parse"
 	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/cmd/podman/report"
```

Comment:
might be part of it, the report pkg was move into containers/common

try rebasing, running a `make vendor-in-container` and see if these pop up/use these imports instead:
```
"github.com/containers/common/pkg/report"
"github.com/containers/podman/v2/cmd/podman/common"
```

Reply:
@ashley-cui @ParkerVR the PR I am working  on now will remove github.com/containers/podman/v2/cmd/podman/report